e2e/stress/IoTClientPerf/Scenarios/ServiceClientScenarios.cs

```diff
@@ -102,11 +103,17 @@ protected async Task CallMethodAsync(CancellationToken ct)
             try
             {
                 var methodCall = new CloudToDeviceMethod(TestMethodName);
-                Task t = s_sc.InvokeDeviceMethodAsync(Configuration.Stress.GetDeviceNameById(_id, _authType), methodCall);
+                Task<CloudToDeviceMethodResult> t = s_sc.InvokeDeviceMethodAsync(Configuration.Stress.GetDeviceNameById(_id, _authType), methodCall);
                 _mMethod.ScheduleTime = _swMethod.ElapsedMilliseconds;
 
                 _swMethod.Restart();
-                await t.ConfigureAwait(false);
+                CloudToDeviceMethodResult result = await t.ConfigureAwait(false);
```

Comment:
Please use `finally`.
```
try
{
  ...
  if (result.Status != MethodPassStatus) 
  {
     _mMethod.ErrorMessage = $"Method Failed - Status: {result.Status} Payload:{result.GetPayloadAsJson()}";
  }
}
catch(Exception ex)
{
  _mMethod.ErrorMessage = $"{ex.GetType().Name} - {ex.Message}";
  throw;
}
finally
{
  _mMethod.ExecuteTime = _swMethod.ElapsedMilliseconds;
   await _writer.WriteAsync(_mMethod).ConfigureAwait(false);
}
```

Reply:
Corrected here and in DeviceScenarios

Reply:
Remove `InvalidOperationException` that is thrown and caught. Replace message with the same pattern but with a non-exception for error title.

Reply:
@davilu Can't remove the exception: the aggregate reports are based on the Task completing, failing or getting cancelled.

Changed all the other items.